pa/fin/tests/index/test_parsers.py

```diff
@@ -0,0 +1,27 @@
+"""
+Tests for Index parsers
+"""
+from random import choice
+
+from fin.models.index import Index
+from fin.models.index.parsers import SlickChartsParser
+from fin.tests.base import BaseTestCase
+
+
+class IndexParsersTests(BaseTestCase):
+    """
+    Tests for Index parsers
+    """
+    def test_slick_charts_parser(self):
+        """
+        Tests the parsing for SlickCharts data source
+        """
+        data_sources = [url for url, parser in Index.url_parsers.items()
```

Comment:
I do not check the code, but seems here can be used the factory pattern to avoid checking the type of the object.